course/layouts/_default/baseof.html

```diff
@@ -81,7 +81,7 @@
 
```

Comment:
gh won't let me comment on it, but the line above this one contains

```
{{- $mathjax := .Site.Data.webpack.mathjax -}}
```

and it doesn't look like we're using that variable anywhere